web/src/components/troubleshoot/SupportBundleAnalysis.jsx

```diff
@@ -222,6 +252,48 @@ export class SupportBundleAnalysis extends React.Component {
       );
     }
 
+    // TODO: make this into a reusable component
+    let progressBar;
```

Comment:
preexisting code to render the progress bar. for now, just making things work.
Will be refactored and made into a reusable component since a few places uses the same logic.